content/manuals/build/buildkit/dockerfile-release-notes.md

```diff
@@ -305,7 +305,7 @@ The following features have graduated from the labs channel to stable:
 
 {{< release-date date="2023-01-10" >}}
 
-{{% include "dockerfile-labs-channel.md" %}}
+{{< summary-bar feature_name="Dockerfile experimental lab" >}}
```

Comment:
We should have this block for all labs releases. Currently missing for 1.14.0 (labs), 1.13.0 (labs) and 1.11.0 (labs)